src/Debugger-Tests/AssignmentAndLiteralDebuggerTest.class.st

```diff
@@ -20,7 +20,7 @@ AssignmentAndLiteralDebuggerTest >> testInterruptedContextEqualsSuspendedContext
 
 { #category : #tests }
 AssignmentAndLiteralDebuggerTest >> testNewDebugSession [
-	self assert: context size equals: 0.
+	self assert: context size equals: 1.
```

Comment:
simulated process must have at least two context: the top one and the given process block (what #newProcess does).